pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -454,6 +455,11 @@ func setupKernelTunables(option KernelTunableBehavior) error {
 			klog.V(2).InfoS("Updating kernel flag", "flag", flag, "expectedValue", expectedValue, "actualValue", val)
 			err = sysctl.SetSysctl(flag, expectedValue)
 			if err != nil {
+				if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.KubeletInUserNamespace) && libcontainersystem.RunningInUserNS() {
+					klog.V(2).InfoS("Updating kernel flag failed (running in UserNS, ignoring)", "flag", flag, "err", err)
+					continue
+				}
+				klog.ErrorS(err, "Updating kernel flag failed (Hint: enable KubeletInUserNamespace feature flag to ignore the error if running in a user namespace)", "flag", flag)
```

Comment:
You can check whether running in user namespace, so no need to output hint in case kubelet running outside of it, right?

Reply:
updated